config/routes.rb

```diff
@@ -24,6 +24,7 @@
 
     resources :cards do
       get 'get_flickr_images', on: :collection
+      post 'parsing_html', on: :collection
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.